instructor/ui/assignment_setup.css

```diff
@@ -125,12 +125,24 @@ select:disabled {
     left: 28px;
 }
 .scb_s_delete_strain {
-    border: 0px;
+    border: 0;
     width: 24px;
     background: transparent;
     height: 20px;
     cursor: pointer;
-    top: 0px;
+    top: 0;
     position: relative;
     left: 16px;
 }
+
+.checkbox_image{
```

Comment:
seems like this class name should be more specific to the use. How about `delete_checkbox_image` ?